### DISCUSSION TITLE
Can't find the Cheat Option in the RA menu.

### DISCUSSION THREAD

**Discussion by makyo5** [OnionUI/Onion#1548] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.0

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

Sorry, may be there is the cheat in the game. (GBA, GB, GBC, PS, MD)
But I hadn't found the cheats option in the RA menu in game.
Where is the cheats option in the RA menu?

### Steps to Reproduce

1, Open a game in OnionOS.
2, I choose GB, GBC, GBA, NES, PSX one of them.
3, Press Menu + Select Button to call out the RA menu.
4, No "Cheats" option in the menu. Back to upper level menu, the same.

thx.

### Provide any additional context or information that may help us investigate your issue

none

**Comment by Aemiii91** [comment]
Did you try following this guide: https://onionui.github.io/docs/faq#how-do-i-add-game-cheats

**Reply by makyo5** [reply]
Thx, I have referred the doc, but missed the FAQ.

**Comment by kwhoami175-ship-it1** [comment]
Rr
